splink/params.py

```diff
@@ -394,7 +394,13 @@ def adjustment_factor_chart(self):  # pragma: no cover
         else:
             return adjustment_weight_chart_def
 
-    def all_charts_write_html_file(self, filename="splink_charts.html"):
+    def all_charts_write_html_file(self, filename="splink_charts.html", overwrite=False):
```

Comment:
Note this isn't necessary to the PR, but when I wrote my function, I realised it's useful to deal with overwriting in this function